### PR TITLE
added minor typos

### PR DESCRIPTION
**Pull request by NuriaQueralt**
Dear @PPerdomoQ could you review again WP::Phenotype model? I fixed some minor typos I found. Also, could you check the semantics of each entity? there should be consistency WP4::phenotype and WP3. Thank you!

### CODE REVIEW

**Comment by NuriaQueralt** [comment]
Dear @PPerdomoQ I have also removed the class 'gene' described by orphanet to classify the mgi dmd gene because orphanet refers to the Human gene.

**Comment by NuriaQueralt** [comment]
Dear @PPerdomoQ could you review syntax and semantics for wp4 molecular component of the model? this is a major update.

**Comment by NuriaQueralt** [comment]
Dear @PPerdomoQ could you review syntax and semantics for wp4::mouse component of the model? this is a major update.

File: fair-data-models/therapeutic-wp4/docs/molecular_outcome.md [hunk 1]
```diff
@@ -109,69 +127,112 @@ PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
...
     a [sio:SIO_000074] ;
     rdfs:label xsd:string?
 }
 
+:genotypeShape IRI {
+    a [obo:SO_0001027] ;
+    a [sio:SIO_000614] ;
+    rdfs:label xsd:string? ;
+    sio:SIO_000300 xsd:string ;
+    sio:SIO_001403 @:isoformShape
```
Comment (PPerdomoQ) [inline fair-data-models/therapeutic-wp4/docs/molecular_outcome.md:159]: Here I would say you need to add a '+' at the end

File: fair-data-models/therapeutic-wp4/docs/molecular_outcome.md [hunk 2]
```diff
@@ -109,69 +127,112 @@ PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
...
+}
+
+:isoformShape IRI {
+    a [obo:~] ;
```
Comment (PPerdomoQ) [inline fair-data-models/therapeutic-wp4/docs/molecular_outcome.md:163]: You can include the list of isoforms in the PR ontology (see above)

File: fair-data-models/therapeutic-wp4/docs/molecular_outcome.md [hunk 3]
```diff
@@ -109,69 +127,112 @@ PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
...
+    sio:SIO_000230 @:treatedInputShape ;
     sio:SIO_000291 @:treatedTargetShape ;
     sio:SIO_000229 @:restorationOutputShape ;
     sio:SIO_000008 @:measurementTimeShape
 }
 
-:treatedTargetShape IRI {
-    a [obo:UBERON_0002037] ;
+:treatedInputShape IRI {
+    a [obo:UBERON_~] ;
```
Comment (PPerdomoQ) [inline fair-data-models/therapeutic-wp4/docs/molecular_outcome.md:216]: I add here the mapping of hippocampus, cortex and cerebellum to facilitate the work :) (I think those are the only anatomical areas in WP4)

[obo:UBERON_0001950] OR [obo:UBERON_0002421] OR [obo:UBERON_0002037]

File: fair-data-models/therapeutic-wp4/docs/molecular_outcome.md [hunk 4]
```diff
@@ -109,69 +127,112 @@ PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
...
     a [obo:UBERON_0002616] ;
     a [sio:SIO_010046] ;
     rdfs:label xsd:string? ;
-    sio:SIO_000068 @:entityShape
+    sio:SIO_000068 @:mouseShape
+}
+
+:treatedTargetShape IRI {
+    a [obo:~] ;
```
Comment (PPerdomoQ) [inline fair-data-models/therapeutic-wp4/docs/molecular_outcome.md:224]: You can add it from the comment above :)

File: fair-data-models/therapeutic-wp4/docs/behavioral_outcome.md [hunk 5]
```diff
@@ -107,101 +125,120 @@ PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
...
+    rdfs:label xsd:string? ;
+    rdfs:comment xsd:string ;
+    sio:SIO_000229 @:behavioralOutputShape ;
+    sio:SIO_000008 @:measurementTimeShape
 }
 
 :behavioralOutputShape IRI {
-    a [obo:CMO_0001043] ;
-    a [obo:IAO_0000109] ;
+    a [obo:~] ;
```
Comment (PPerdomoQ) [inline fair-data-models/therapeutic-wp4/docs/behavioral_outcome.md:214]: I think you can use something like: obo:CMO_0001043. What do you think?

File: fair-data-models/therapeutic-wp4/docs/mouse_outcome.md [hunk 6]
```diff
@@ -91,72 +88,64 @@ PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
...
     rdfs:label xsd:string? ;
-    sio:SIO_000300 xsd:string
-}
-
-:treatedTargetShape IRI {
-    a [obo:UBERON_0002616] ;
-    a [sio:SIO_010046] ;
-    rdfs:label xsd:string? ;
-    sio:SIO_000068 @:entityShape
+    sio:SIO_001403 @:isoformShape
```
Comment (PPerdomoQ) [inline fair-data-models/therapeutic-wp4/docs/mouse_outcome.md:138]: Same as above. I would say a '+' is needed at the end

File: fair-data-models/therapeutic-wp4/docs/mouse_outcome.md [hunk 7]
```diff
@@ -91,72 +88,64 @@ PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
...
 }
 
-:treatedAttributeShape IRI {
-    a [obo:PR_P11531-1] ;
-    a [sio:SIO_000614] ;
+:isoformShape IRI {
+    a [obo:~] ;
```
Comment (PPerdomoQ) [inline fair-data-models/therapeutic-wp4/docs/mouse_outcome.md:142]: You can copy the isoforms from the coment above :)

**Comment by PPerdomoQ** [comment]
Hi @NuriaQueralt ! i finished the first revision of your changes and they look really good :) I just added a few comments that I hope can help you to facilitate your work!

File: fair-data-models/therapeutic-wp4/docs/behavioral_outcome.md [hunk 8]
```diff
@@ -107,101 +125,120 @@ PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
...
     a [obo:SO_0001027] ;
     a [sio:SIO_000614] ;
     rdfs:label xsd:string? ;
-    sio:SIO_000300 xsd:string
+    sio:SIO_000300 xsd:string ;
+    sio:SIO_001403 @:isoformShape
+}
+
+:isoformShape IRI {
+    a [obo:~] ;
```
[the remark this answers is not in the document]
Comment (NuriaQueralt) [reply fair-data-models/therapeutic-wp4/docs/behavioral_outcome.md:162]: fixed

File: fair-data-models/therapeutic-wp4/docs/molecular_outcome.md (hunk 1 above)
Comment (NuriaQueralt) [reply fair-data-models/therapeutic-wp4/docs/molecular_outcome.md]: fixed

File: fair-data-models/therapeutic-wp4/docs/molecular_outcome.md (hunk 2 above)
Comment (NuriaQueralt) [reply fair-data-models/therapeutic-wp4/docs/molecular_outcome.md]: fixed

File: fair-data-models/therapeutic-wp4/docs/molecular_outcome.md (hunk 4 above)
Comment (NuriaQueralt) [reply fair-data-models/therapeutic-wp4/docs/molecular_outcome.md]: fixed

File: fair-data-models/therapeutic-wp4/docs/mouse_outcome.md (hunk 6 above)
Comment (NuriaQueralt) [reply fair-data-models/therapeutic-wp4/docs/mouse_outcome.md]: fixed

File: fair-data-models/therapeutic-wp4/docs/mouse_outcome.md (hunk 7 above)
Comment (NuriaQueralt) [reply fair-data-models/therapeutic-wp4/docs/mouse_outcome.md]: fixed

**Review by NuriaQueralt — COMMENTED** [review]
Dear @PPerdomoQ, thank you so much for your careful review and for facilitating my life adding all the terms needed instead of the regular expressions. I fixed all your suggestions. You should be able to accept and do the merge.

**Pull request merged**